### PR TITLE
NXP-30182: Keep a bulk status for 24h in case of error

### PR DESCRIPTION
**Pull request by bdelbosc**

### CODE REVIEW

**Review by kevinleturc — APPROVED** [review]

File: modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkServiceImpl.java [hunk 1]
```diff
@@ -99,7 +99,11 @@
     public static final long COMPLETED_TTL_SECONDS = 3_600;
 
     // How long we keep the command and its status in the kv store once aborted
-    public static final long ABORTED_TTL_SECONDS = 7_200;
+    public static final long ABORTED_TTL_SECONDS = 43_200;
+
+    // @since 11.5
+    // How long we keep the command and its status in the kv store once completed with an error
+    public static final long COMPLETED_IN_ERROR_TTL_SECONDS = 86_400;
```
Comment (troger) [inline modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkServiceImpl.java:106]: Worth using `Duration` here to get something easier to read? :)

Something like
```
public static final long ABORTED_TTL_SECONDS =  Duration.ofHours(1).toSeconds()
```
?

Comment (bdelbosc) [reply modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkServiceImpl.java:106]: 86_400 is 2 times easier than 43_200, but using duration will make it hard to backport in java8 :)

Comment (efge) [reply modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkServiceImpl.java:106]: Duration is in Java 8 :)

Comment (bdelbosc) [reply modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkServiceImpl.java:106]: yes but with different methods getSeconds vs toSeconds

Comment (efge) [reply modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkServiceImpl.java:106]: Yes toSeconds was added after 8.

Comment (efge) [reply modules/core/nuxeo-core-bulk/src/main/java/org/nuxeo/ecm/core/bulk/BulkServiceImpl.java:106]: But getSeconds still exists in 11 and does the same thing.

**Review by troger — APPROVED** [review]

**Pull request merged**